backend/api/src/test/kotlin/software/shonk/adapters/incoming/ShorkInterpreterControllerV1IT.kt

```diff
@@ -207,4 +225,48 @@ class ShorkInterpreterControllerV1IT() : AbstractControllerTest() {
         assertEquals(HttpStatusCode.Created, result3.status)
         assertEquals("3", result3.bodyAsText())
     }
+
+    @Test
+    fun `test get all lobbies when there is only one and it's not initialized`() = runTest {
+        val result = client.get("/api/v1/lobbies")
+        val parsedLobbies = parseAllLobbies(result)
+
+        assertEquals(HttpStatusCode.OK, result.status)
+        assertEquals(1, parsedLobbies.size)
+        assertNotEquals("playerA,playerB", parsedLobbies[0]["playersJoined"])
+        assertEquals("NOT_STARTED", parsedLobbies[0]["gamestate"])
+    }
+
+    @Test
+    fun `test get all lobbies as a list with players and games status for more than one lobby`() =
+        runTest {
+            client.post("/api/v1/lobby/0/code/playerA") {
+                contentType(ContentType.Application.Json)
+                setBody("someString")
+            }
```

Comment:
this will fail in the future, since the "default lobby with id 0" is to be removed from apiV1. Please have another look in the discord channel and ESPECIALLY the architecture diagram. It should explain exactly that.

Reply:
I changed the initial parameter of the lobbyId to the 0 so that it can be checked if it's null and doesn't have to be updated when apiv1 gets updated and the default lobby gets removed. Only the tests have to be updated then

Reply:
this issue was resolved in the logic in getAllLobbies already by not relying on any lobby id when adding a new lobby to the list